Sources/MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -85,6 +98,21 @@ open class PassiveLocationDataSource: NSObject {
             }
         }
     }
+
+    /**
+     Sets electronic horizon options. Pass `nil` to reset to defaults.
+     */
+    public func set(electronicHorizonOptions: ElectronicHorizonOptions?) {
```

Comment:
The public API should use a pure Swift struct instead of ElectronicHorizonOptions. It would be converted to or from an ElectronicHorizonOptions on the fly.

Reply:
Agree, adjusted for that.